com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/HDTarget.cs

```diff
@@ -595,12 +595,17 @@ static class CoreRequiredFields
     {
         public static FieldCollection Meta = new FieldCollection()
         {
+            HDStructFields.AttributesMesh.positionOS,
             HDStructFields.AttributesMesh.normalOS,
-            HDStructFields.AttributesMesh.tangentOS,
             HDStructFields.AttributesMesh.uv0,
             HDStructFields.AttributesMesh.uv1,
-            HDStructFields.AttributesMesh.color,
             HDStructFields.AttributesMesh.uv2,
+            HDStructFields.AttributesMesh.uv3,
+            HDStructFields.FragInputs.positionRWS,
+            HDStructFields.FragInputs.texCoord0,
+            HDStructFields.FragInputs.texCoord1,
+            HDStructFields.FragInputs.texCoord2,
+            HDStructFields.FragInputs.texCoord3,
```

Comment:
can remove texcoords since they will be added when corresponding uvs are used

Reply:
If I remove these, the incorrect UVs are used to sample the debug texture, so I am leaving them in:
![image](https://user-images.githubusercontent.com/3450690/120662952-6cf5da00-c43e-11eb-9cd0-e466974a6dd2.png)